### ISSUE TITLE
Use Markdown

### ISSUE THREAD

**Issue by Lucretia** [Ada-Rapporteur-Group/User-Community-Input#11]
Hi,

Use MD and convert using an existing application like AsciiDoc(tor)/Pandoc to convert to the necessary format (including RTF - see pandoc).

Luke.

**Comment by Lucretia** [comment]
👍

**Comment by CKWG** [comment]
This is direly needed. Currenty when opening an AI on /Active AIS/, you just get unreadable junk.

**Comment by ARG-Editor** [comment]
Not really. The Github here is still experimental and not maintained. If you want to see AIs (especially as they are intended to be seen), use the Ada-Auth.org site or (for newer AIs), directly view the Google Docs (accessible from the arg.adaic.org site).

Simplistic markups like Markdown can't support the full complexity of formatting used in the Reference Manual (and thus AIs), something that HTML can do easily. So Markdown AI versions will never be the official versions - it would be silly to lose much of the formatting. They're similar to the plain text versions of the RM, which have uses but are not usable as the sole reference.

**Comment by sttaft** [comment]
On Fri, Sep 2, 2022 at 4:21 AM ARG-Editor ***@***.***> wrote:

> Not really. The Github here is still experimental and not maintained. If
> you want to see AIs (especially as they are intended to be seen), use the
> Ada-Auth.org site or (for newer AIs), directly view the Google Docs
> (accessible from the arg.adaic.org site). ...
>

This is a bit misleading.  The *issues* on the ARG GitHub are not
experimental any more.  It is only the Active_AIs directory that you should
ignore (and should be deleted at this point to avoid confusion).

-Tuck

**Comment by ARG-Editor** [comment]
Sorry. I meant the Ada Issues (AIs) stored in the Github version control are experimental and not maintained. The Github Issues section of course is fully active.

**Comment by Lucretia** [comment]
> Simplistic markups like Markdown can't support the full complexity of formatting used in the Reference Manual (and thus AIs), something that HTML can do easily.

It can as you can embed HTML directly into MD files.
